spec/views/rb_taskboards/show_spec.rb

```diff
@@ -224,7 +229,7 @@
 
       render
 
-      assert_select ".model.work_package.task" do |task|
+      assert_select '.model.work_package.task' do |task|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.